.github/scripts/trymerge.py

```diff
@@ -981,6 +983,28 @@ def merge_into(self, repo: GitRepo, *,
             for pr in additional_merged_prs:
                 pr.add_numbered_label("merged")
 
+        if comment_id and self.pr_num:
+            # Finally, upload the record to Rockset. The list of pending and failed
+            # checks are at the time of the merge
+            save_merge_record(
+                collection=ROCKSET_MERGES_COLLECTION,
```

Comment:
I would like to petition to add the commit_sha if the pr successfully merges

I would also like to petition that it doesnt save dry runs, so that local testing doesnt add a ton of things to the rockset collection (not super important tho, i imagine that the check for comment_id solves this usually)

It's also worth noting that the interior prs in a ghstack wouldn't get entries, but I'm not sure how important that is

Reply:
Yeah, ghstack would only get only entry for the top PR, but I would rather deal with that later when there is a need.  At t least, the top of the stack will still have a record.

Reply:
> I would also like to petition that it doesnt save dry runs, so that local testing doesnt add a ton of things to the rockset collection (not super important tho, i imagine that the check for comment_id solves this usually)

I guess what you mean is dry-run won't save anything into Rockset.  This can be done.  I will keep it just for testing this change, and will add a check to skip uploading to Rockset when dry-run is set.

The commit SHA, base SHA are important.  So let me add that.